test/inmem/sa/sa.go

```diff
@@ -131,3 +132,48 @@ func (sa SA) AddBlockedKey(ctx context.Context, req *sapb.AddBlockedKeyRequest,
 func (sa SA) FQDNSetExists(ctx context.Context, req *sapb.FQDNSetExistsRequest, _ ...grpc.CallOption) (*sapb.Exists, error) {
 	return sa.Impl.FQDNSetExists(ctx, req)
 }
+
+type mockSerialForIncidentStream_Result struct {
```

Comment:
nit, here and in the other mock type names:
```suggestion
type mockSerialsForIncidentStream_Result struct {
```